client/visualizer/webpack.config.js

```diff
@@ -27,6 +28,26 @@ var conf = {
       { test: /\.css$/, loader: "style-loader!css-loader" }
     ]
   },
+  plugins: [
+    new CopyWebpackPlugin([
+      {
+        from: path.resolve(__dirname, 'node_modules/speedscope/dist/release'),
+        to: path.resolve(__dirname, 'bc20/speedscope'),
+        transform: (content, filePath) => {
+          // Make speedscope's localProfilePath hash parameter support relative paths
+          if (filePath.endsWith('.js')) {
+            return content.toString().replace('file:///', '');
+          }
+
+          return content;
+        }
+      }
+    ])
+  ],
+  devServer: {
+    // Required to ensure the files copied by the CopyWebpackPlugin are copied when running the dev server
+    writeToDisk: filePath => filePath.includes('speedscope/')
+  },
```

Comment:
hmm why is this necessary

Reply:
When using webpack-dev-server (with `npm run watch` in the client directory) the CopyWebpackPlugin doesn't actually copy the speedscope files without this line because webpack-dev-server does everything in-memory. This causes problems because those speedscope files are needed by the iframe visible when the Profiler tab is active. By using `writeToDisk` we can tell webpack-dev-server to write certain files to disk even when running primarily in-memory (in this case all the speedscope files).

Reply:
i see i see